src/Repository/UserRepository.php

```diff
@@ -112,19 +112,21 @@ public function lookUpByEmail(string $email) {
      */
     public function findLatestContributions(User $user, int $limit = 25) {
         $sql = <<<EOSQL
-SELECT JSON_AGG(id) AS ids, type FROM (
-        SELECT id, timestamp, 'comment'::TEXT AS type FROM comments WHERE user_id = :user_id
+SELECT JSON_AGG(id) AS ids, entry_type FROM (
+        SELECT c.id, c.timestamp, 'comment'::TEXT AS entry_type FROM comments c
+            INNER JOIN submissions s on c.submission_id = s.id
+        WHERE c.user_id = :user_id and s.mod_thread = false
     UNION ALL
-        SELECT id, timestamp, 'submission'::TEXT AS type FROM submissions WHERE user_id = :user_id
+        SELECT id, timestamp, 'submission'::TEXT AS entry_type FROM submissions WHERE user_id = :user_id AND mod_thread = false
     ORDER BY timestamp DESC
```

Comment:
@NathanDeckinga 
It's using timestamp to order them properly. When I remove it from the select, it breaks, so I need to change the sort as well.

I could change it to sort by IDs, but then it'd be a mix of submission ids and comment ids right? Which wouldn't give us proper sorting.

Suggestions?

Reply:
I missed that earlier - you are absolutely correct that it is.  Ignore my earlier comment about it not using the limit as well, I didn't make it down to this part of the union statement.  I think you are completely good then.